sematic/graph.py

```diff
@@ -448,7 +449,17 @@ def _clone_run(
 
         func = run.get_func()
 
-        future = func(**kwargs)
+        if run.calculator_path == "sematic.calculator._make_list":
```

Comment:
Please add a code comment that this is necessary due to the `calculator_path` resolving to the function that generates the `Calculator`, not to an annotated function.

Reply:
Done.